tests/test_extraction/test_text_extractor.py

```diff
@@ -0,0 +1,197 @@
+"""
+    This script is for unit testing of text_extractor
+
+    Use pytest to run this script
+    Command to run: /stampify$ python -m pytest
+"""
+import bs4
+import pytest
+
+from extraction import text_extractor
+from extraction.data_models.text import Text
+from extraction.utils import string_utils as utils
+
+__EXTRACTOR = text_extractor.TextExtractor()
+
+
+@pytest.fixture(scope='module')
+def soup():
```

Comment:
**TODO:** Move to test_utils in next iterations.

Reply:
Add as comment in the code.

Reply:
Added! Thanks for the suggestion.